docs/how_to/create_a_map.rst

```diff
@@ -0,0 +1,99 @@
+.. _how-to-create-a-map:
+
+How to create a sunpy Map
+=========================
+
+One of the primary goals of the Map interface is to make it as easy as possible to create a Map.
+As such, you can pass many different kinds of inputs to Map.
+These are enumerated below.
+
+File name
+---------
+
+This is the easiest and recommended way to create a Map. This can be either a string or a `pathlib.Path`.
+
+.. code-block:: python
+
+    >>> import sunpy.map
+    >>> import sunpy.data.sample
+    >>> my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)  # doctest: +REMOTE_DATA
+    >>> my_map = sunpy.map.Map('file.fits')  # doctest: +SKIP
+    >>> import pathlib
+    >>> my_map = sunpy.map.Map(pathlib.Path('file.fits'))  # doctest: +SKIP
+    >>> sub_dir = pathlib.Path('local_dir/sub_dir')
+    >>> mymap = sunpy.map.Map(sub_dir / 'another_file.fits')   # doctest: +SKIP
+
+Data array and an `astropy.io.fits.Header`
+------------------------------------------
+
+.. code-block:: python
+
+    >>> import astropy.io.fits
+    >>> with astropy.io.fits.open(sunpy.data.sample.AIA_171_IMAGE) as hdul:
+    ...     data = hdul[1].data
+    ...     header = hdul[1].header  # doctest: +REMOTE_DATA
+    >>> my_map = sunpy.map.Map(data, header)  # doctest: +REMOTE_DATA
```

Comment:
```suggestion
    >>> import astropy.io.fits

    >>> with astropy.io.fits.open(sunpy.data.sample.AIA_171_IMAGE) as hdul:
    ...     data = hdul[1].data
    ...     header = hdul[1].header  # doctest: +REMOTE_DATA
    >>> my_map = sunpy.map.Map(data, header)  # doctest: +REMOTE_DATA
```

Reply:
As mentioned above, I don't see what the addded spacing adds.